build/cmake/options.cmake

```diff
@@ -36,15 +36,17 @@ endif()
 
 if(MSVC)
     wx_option(wxBUILD_MSVC_MULTIPROC "Enable multi-processor compilation for MSVC")
-else()
-    # Other compilers support setting the C++ standard, present it an option to the user
+endif()
+
+if(NOT MSVC OR NOT MSVC_VERSION GREATER 1900)
```

Comment:
"**OR NOT** MSVC_VERSION GREATER 1900" Does the "NOT" really belong there?

Additionally, MSVC does not support all C++ standards listed below (only 14, 17, and latest) but I guess this applies to other compilers as well...

Reply:
That's indeed wrong. I wanted to change it to `NOT MSVC_VERSION LESS 1900` but I guess I'll just keep `MSVC_VERSION GREATER 1800` for simplicity (`GREATER_EQUAL` is not available in older CMake versions).